### PR TITLE
Enable BraveVPN for Windows/macOS/Android 5% (production)

### PR DESCRIPTION
**Pull request by bsclifton**
Enable `BraveVPN` feature on **Windows**, **macOS**, and **Android** (brave://flags/#brave-vpn)

Enables for 5% of folks using PRODUCTION brave-variation servers

Addresses https://github.com/brave/brave-browser/issues/25680

### CODE REVIEW

**Comment by bsclifton** [comment]
NOTE: should be merged AFTER https://github.com/brave/brave-variations/pull/536

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on `main` (staging) as per https://github.com/brave/brave-variations/pull/457#issuecomment-1462811197, https://github.com/brave/brave-variations/pull/457#issuecomment-1462861073, https://github.com/brave/brave-variations/pull/457#issuecomment-1462892777https://github.com/brave/brave-variations/pull/457#issuecomment-1463921918.

**Pull request merged**
